pom.xml

```diff
@@ -60,6 +60,7 @@
         <version.lib.groovy-all>2.4.14</version.lib.groovy-all>
         <!-- Silence javadoc error org.jboss.logging.annotations.Message$Format not found -->
         <version.lib.jboss-logging-annotations>2.2.1.Final</version.lib.jboss-logging-annotations>
+        <version.lib.jsoup>1.15.3</version.lib.jsoup>
```

Comment:
Using this location to just add a general comment.

My main recommendation for this change is to add more documentation, a README.md would be ideal that showcases the problem statement, approach taken, and finally usage pattern.